### PR TITLE
feat : made the site responsive

### PR DESCRIPTION
**Pull request by BalajiHarinath**
I have made the site responsive. Please review. (Please have a look at the media queries in css files)

### CODE REVIEW

File: css/global.css [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
     --text-color1 : #3b3939;
     --border-color1 : #b4b1b1;
-    --background-color1 : #eaeaec99;
+    --background-color1 : hsla(240, 5%, 92%, 0.6);
+    --background-color2 : hsl(240, 5%, 92%);
     --box-shadow : 1px 1px 10px 1px #3b3f4266;
```
Comment (Anurag021) [inline css/global.css:11]: if possible try to use same color format, i.e. either hsl or hex.

**Review by Anurag021 — APPROVED** [review]
Just one suggestion given above, rest all code looks good to me.

File: css/global.css (hunk 1 above)
Comment (BalajiHarinath) [reply css/global.css:11]: Yes I actually missed to change this. Thanks for pointing this out . Will fix it

File: Pages/Product page/product.css [hunk 2]
```diff
@@ -205,31 +209,116 @@
...
     :root{
-        font-size: 12px;
+        font-size: 14px;
     }
 }
 
-@media screen and (max-width : 850px) {
-    :root{
-        font-size: 10px;
+@media screen and (max-width : 880px) {
```
Comment (sainath7878) [inline Pages/Product page/product.css]: This can be merged in 850px media queries.

Comment (BalajiHarinath) [reply Pages/Product page/product.css]: Yes will update this. Thanks for reviewing.

**Pull request merged**